tests/eikonal_equation_01.cc

```diff
@@ -0,0 +1,30 @@
+#include "pidomus.h"
+#include "interfaces/eikonal_equation.h"
+#include "tests.h"
+
+using namespace dealii;
+int main (int argc, char *argv[])
+{
+
+  Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv,
+                                                      numbers::invalid_unsigned_int);
+
+  initlog();
+  deallog.depth_file(1);
+
+  const int dim = 2;
+  const int spacedim = 2;
+
+  EikonalEquation<dim,spacedim,LATrilinos> p;
+  piDoMUS<dim,spacedim,LATrilinos> solver ("pidomus", p);
+  ParameterAcceptor::initialize(SOURCE_DIR "/parameters/eikonal_equation_01.prm", "used_parameters.prm");
+
+
+  solver.run ();
+
+  auto sol = solver.get_solution();
+  for (unsigned int i = 0; i<sol.size(); ++i)
```

Comment:
Capito… :)
Grazie!

> On 10 May 2016, at 16:38, Alberto Sartori <notifications@github.com> wrote:
> 
> In tests/eikonal_equation_01.cc <https://github.com/mathLab/pi-DoMUS/pull/191#discussion_r62683348>:
> 
> > +
> > +  initlog();
> > +  deallog.depth_file(1);
> > +
> > +  const int dim = 2;
> > +  const int spacedim = 2;
> > +
> > +  EikonalEquation<dim,spacedim,LATrilinos> p;
> > +  piDoMUS<dim,spacedim,LATrilinos> solver ("pidomus", p);
> > +  ParameterAcceptor::initialize(SOURCE_DIR "/parameters/eikonal_equation_01.prm", "used_parameters.prm");
> > +
> > +
> > +  solver.run ();
> > +
> > +  auto sol = solver.get_solution();
> > +  for (unsigned int i = 0; i<sol.size(); ++i)
> @ESeNonFossiIo <https://github.com/ESeNonFossiIo> questo è il problema se vai in mpi..
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/mathLab/pi-DoMUS/pull/191/files/5be2bc6d41cf4b5a15603c05413f966df9f4f5ae#r62683348>